sympy/integrals/laplace.py

```diff
@@ -997,6 +1001,41 @@ def _laplace_apply_simple_rules(f, t, s):
     return None
 
 
+def _piecewise_to_heaviside(f, t):
+    """
+    This function converts a Piecewise expression to an expression written
+    with Heaviside. It is not exact, but valid in the context of the Laplace
+    transform.
+    """
+    x = piecewise_exclusive(f)
+    r = S.Zero
+    for term in x.args:
+        # Here we do not need to do many checks because piecewise_exclusive
+        # has a clearly predictable output. However, if any of the conditions
+        # is not relative to t, this function just returns the input argument.
+        fn = term[0]
+        op = term[1].func
+        if op == LessThan or op == StrictLessThan:
+            if not term[1].args[0] == t:
+                return f
+            right_boundary = term[1].args[1]
+            r = r + Heaviside(-(t-right_boundary))*fn
+        if op == GreaterThan or op == StrictGreaterThan:
+            if not term[1].args[0] == t:
+                return f
+            left_boundary = term[1].args[1]
+            r = r + Heaviside(t-left_boundary)*fn
+        if op == And:
+            if not (
+                    term[1].args[0].args[0] == t and
+                    term[1].args[1].args[0] == t):
+                return f
+            left_boundary = term[1].args[0].args[1]
+            right_boundary = term[1].args[1].args[1]
+            r = r + (Heaviside(t-right_boundary)-Heaviside(t-left_boundary))*fn
```

Comment:
```suggestion
    for fn, cond in x.args:
        # Here we do not need to do many checks because piecewise_exclusive
        # has a clearly predictable output. However, if any of the conditions
        # is not relative to t, this function just returns the input argument.
        if isinstance(cond, Relational) and t in cond.args:
			r = r + Heaviside(cond.gts - cond.lts)*fn
        elif isinstance(cond, And) and len(cond.args) == 2 and cond.args[0].gts == cond.args[1].lts == t:
            c1, c2 = cond.args  # And(a < t, t < b)
            r = r + (Heaviside(t - c2.gts) - Heaviside(t - c1.lts))*fn
		else:
		    return f
```

Reply:
The import of `LessThan, StrictLessThan, GreaterThan, StrictGreaterThan` would need to be changed to `Relational`, too.

Reply:
This is not quite correct, but beautiful, thanks!

I'll check in the correct code in a minute.